exporter/importables.go

```diff
@@ -953,17 +953,36 @@ var resourcesMap map[string]importable = map[string]importable{
 		Name: func(d *schema.ResourceData) string {
 			return globalWorkspaceConfName
 		},
+		List: func(ic *importContext) error {
+			ic.Emit(&resource{
+				Resource: "databricks_workspace_conf",
+				ID:       globalWorkspaceConfName,
+				Data: ic.Resources["databricks_workspace_conf"].Data(
+					&terraform.InstanceState{
+						ID:         globalWorkspaceConfName,
+						Attributes: map[string]string{},
+					}),
```

Comment:
It's not strictly required to explicitly create empty `r.Data` - it will be created automatically: https://github.com/databricks/terraform-provider-databricks/blob/master/exporter/context.go#L441. You may need to provide `InstanceState` only when you're initializing it

Reply:
@alexott In the implementation of the List: func
I've removed the unnecessary Data: block and pushed the commit after testing it.